docs-src/typescript/versioning.md

```diff
@@ -0,0 +1,224 @@
+---
+id: versioning
+title: Versioning
+description: Versioning lets you update Workflow Definitions without disrupting current long-running Workflows.
+sidebar_label: Versioning
+tags:
+  - guide-context
+---
+
+Versioning (known as "patching" in TypeScript) lets you update Workflow Definitions without disrupting current long-running Workflows.
```

Comment:
Is this the new sentence?
Not sure what "disrupting" means here - 
Is it to avoid a non-deterministic error?

Reply:
@flossypurse: Yes, this sentence is new. I think that "interrupting" would be better than "disrupting" in this context. (If we want to get more specific, the sentence could read "...without introducing non-deterministic errors in current....")